src/main.jsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from "react";
```

Comment:
Great implementation of react-query to get and then use your data in your application!
While this is impressive, it is important to understand how to implement the features using base state!

Well done though this is clean code